app/web/readme.md

```diff
@@ -1,16 +1,73 @@
 # Couchers web frontend
 
-[![Powered by Vercel](https://www.datocms-assets.com/31049/1618983297-powered-by-vercel.svg)](https://vercel.com?utm_source=couchers-org&utm_campaign=oss)
+This is the react/nextjs web frontend for Couchers.org. We are using Typescript with [React Query](https://react-query.tanstack.com/) for data fetching and [Material UI](https://material-ui.com/) for components.
 
-This is the react/nextjs web frontend for couchers.org. We are using Typescript with [React Query](https://react-query.tanstack.com/) for data fetching and [Material UI](https://material-ui.com/) for components.
+Communication with the backend is via [protobuf messages](https://github.com/protocolbuffers/protobuf-javascript) over [grpc-web](https://github.com/grpc/grpc-web). You can find some helpful documentation on [protobuf messages in javascript here](https://protobuf.dev/protobuf-javascript/).
 
-Communication with the backend is via [protobuf messages](https://github.com/protocolbuffers/protobuf/tree/master/js) and [grpc-web](https://github.com/grpc/grpc-web). You can find some helpful documentation on [protobuf messages in javascript here](https://developers.google.com/protocol-buffers/docs/reference/javascript-generated).
+## Quick Start
+
+We recommend `nvm` (the [node version manager](https://github.com/nvm-sh/nvm)). You can install it with:
```

Comment:
You say `nvm` here but `yarn` below. Usually it's one or the other for a repo. Which is the right one?

Also are we using multiple node versions in the same repo? Otherwise can just be npm.

Reply:
We’re still using yarn. NVM just manages the nodejs version, which is separate from the package manager (npm/yarn) version.

I think nvm is the easiest way to install the appropriate nodejs version? It seems a solid choice for me (with `—no-use`, hence the “recommendation”).

Reply:
You can also do `brew install node`! To do a specific node version you do `brew install node@20.0.00` or whatever. 

Might be easier since it seems nvm causes having to add a lot of tags and stuff.

At least every company I've worked at we used brew. But maybe because you're using linux you gotta Do that. Do you wanna add a Windows heading to yours and I can add a Mac version?

Reply:
lollzzz sorry I meant to say Linux didn't mean to insult you ;-)